src/cascadia/TerminalConnection/ConptyConnection.idl

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.Terminal.TerminalConnection
         Guid Guid { get; };
         String Commandline { get; };
         void ClearBuffer();
+        void ReparentWindow(UInt64 newParent);

```

Comment:
mind filing a followup to make this and the following ones about focus part of a new interface? `ITerminalConnectionWindowHosting` or something